File: api/Entities/Models/UserModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Microsoft.AspNetCore.Mvc.Rendering;


namespace Entities.Models
{
    public class UserModel
    {
        public int Id { get; set; }
        public string? UserName { get; set; }
        public string Email { get; set; } = null!;
        public string? Gender { get; set; }
        public int Languageid { get; set; }
        public int RoleId { get; set; }
        public string Password { get; set; } = null!;
        public DateTime? CreatedOn { get; set; }
        public bool IsDeleted { get; set; }
    }
}

File: ui/QuizProject/src/app/models/user.model.ts
import { ISelectOptions } from "./forms/InputFieldsProps";

export interface UserModel {
    id: number;
    userName: string;
    email: string;
    gender: string;
    languageid: number;
    roleId: number;
    password: string;
    createdOn?: Date;
    isDeleted: boolean;
    userImgPath?: string;
    userScore?: number;
    userImage?: File;
}
